tasks/plugins.yml

```diff
@@ -2,10 +2,16 @@
 
 - name: Get list of installed kibana plugins
   shell: >
-      {{ kibana_bin_dir }}/kibana-plugin list
+      {{ kibana_bin_dir }}/kibana-plugin list {{ _list_plugins_cmd_filter|default('') }}
```

Comment:
I can't seem to find `_list_plugins_cmd_filter` used or defined anywhere else apart from here. Do we need to use it? Or would it be clearer just to call `kibana-plugin list`?

Reply:
Just tested.  Kibana-plugin list would just be sufficient. Have made the change.